docs/BUILD.md

```diff
@@ -112,6 +112,8 @@ s2n-tls has a dependency on a libcrypto library. A supported libcrypto must be l
 - [AWS-LC](https://github.com/aws/aws-lc)
   - Limited ["Sandboxing"](https://github.com/aws/aws-lc/blob/main/SANDBOXING.md) is only supported and tested with AWS-LC.
   - [PQ key exchange](https://aws.github.io/s2n-tls/usage-guide/ch15-post-quantum.html) is only supported with AWS-LC.
+  - s2n-tls enters FIPS mode when linked to AWS-LC-FIPS. AWS-LC-FIPS can be built by first checking out the FIPS tagged
+    commit from the official AWS-LC repo (eg. [AWS-LC-FIPS-2.0.17](https://github.com/aws/aws-lc/tree/AWS-LC-FIPS-2.0.17))
```

Comment:
I know you copied this from your other docs, but I don't think the format / phrasing makes sense in this context. All the other list items are discussing features being supported / not supported. How about:
```suggestion
  - FIPS mode is supported if a FIPS validated version of AWS-LC is used. See the AWS-LC repo for a list of AWS-LC-FIPS tagged commits (or does the AWS-LC repo have any documentation on supporting FIPS you can link to?)
```

Reply:
They have https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/FIPS.md which speaks about their FIPS validations but doesnt cover how to build a FIPS version.

They also have https://github.com/aws/aws-lc/blob/main/BUILDING.md which talks about building the library but doesn't mention how to build a FIPS variant of the library.

I might be missing something so let me double check if there are FIPS specific instructions.